src/main/java/com/nha/abdm/wrapper/hip/hrp/database/mongo/tables/helpers/RequestStatus.java

```diff
@@ -27,6 +27,7 @@ public enum RequestStatus {
   CONSENT_ON_NOTIFY_RESPONSE_RECEIVED("Response received from gateway for hiu notify"),
   CONSENT_FETCH_ACCEPTED("Consent fetch request accepted by gateway"),
   CONSENT_FETCH_ERROR("Error thrown by Gateway for consent fetch"),
+  CONSENT_NOTIFY_ERROR("None of the careContexts present for the requested HI-Type"),
```

Comment:
The error statement and usage does not go along with each other. The error usage is very generic and we dont really know that the message we need to display is exactly "None of the careContexts present for the requested HI-Type"

Reply:
There is a new error code and message Invalid careContexts / no care contexts found, so to simplify the understanding I have given the above message.

Reply:
which comes in the response body of consent/hiu/notify